app-ios/Sources/Model/Entity/EventMapEvent.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public struct EventMapEvent: Equatable, Sendable, Hashable {
+    public let name: MultiLangText
+    public let roomName: MultiLangText
+    public let roomIcon: RoomIcon
+    public let description_: MultiLangText
```

Comment:
I think this `_` isn't needed for iOS side code 👀 

```suggestion
    public let description: MultiLangText
```

Reply:
I've renamed it.